exercises/word-count/example.go

```diff
@@ -19,8 +19,7 @@ func WordCount(phrase string) Frequency {
 }
 
 func normalize(phrase string) string {
-	//  Allow for apostrophes in words.
-	r, _ := regexp.Compile(`[^\w|']`)
 	phrase = strings.ToLower(phrase)
-	return r.ReplaceAllLiteralString(phrase, " ")
+	// allow for apostrophes in words
+	return regexp.MustCompile(`[^\w|']`).ReplaceAllLiteralString(phrase, " ")
```

Comment:
Another `MustCompile` with static regex pattern better be a global or otherwise only called once.